lib/manageiq-api-common.rb

```diff
@@ -1 +1,2 @@
 require 'manageiq/api/common'
+require 'exceptions'
```

Comment:
this file is just used as a redirect to the main require file.  So, this line really belongs in `lib/manageiq/api/common.rb`

Reply:
If the file is properly namespaced, the require shouldn't be necessary since autoload should handle it.